http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -59,19 +62,20 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
   ): F[CreateClusterResult] =
     for {
       ctx <- ev.ask
-      _ <- logger.info(
-        s"Beginning cluster creation for cluster ${params.clusterId} in project ${params.googleProject} | trace id: ${ctx.traceId}"
-      )
 
       // Grab records from the database
       clusterOpt <- kubernetesClusterQuery.getMinimalClusterById(params.clusterId).transaction
       dbCluster <- F.fromOption(
         clusterOpt,
         KubernetesClusterNotFoundException(
-          s"Failed kubernetes cluster creation. Cluster with id ${params.clusterId} not found in database"
+          s"Failed kubernetes cluster creation. Cluster with id ${params.clusterId.id} not found in database | trace id: ${ctx.traceId}"
         )
       )
 
+      _ <- logger.info(
+        s"Beginning cluster creation for cluster ${dbCluster.getGkeClusterId.toString} | trace id: ${ctx.traceId}"
+      )
```

Comment:
Should we say "...for cluster with ID ${dbCluster.getGkeClusterId.toString}..."? Similarly below.

Reply:
`getGkeClusterId.toString` returns a string like `projects/${project.value}/locations/${location.value}/clusters/${clusterName.value}`. I like it because it's pretty imformative, but because of this I don't feel that "with ID" is necessary. (Perhaps `getGkeClusterId` itself should be named differently).

Reply:
Oh right. No need for `with ID` then indeed. 

> Perhaps getGkeClusterId itself should be named differently)

Yea that and `KubernetesClusterId` seem misleading since that's not usually what we mean by `id`s in other places.

Reply:
I feel like there are several places we might want to refactor the Leo/wb-libs k8s models but I'm holding off.. maybe can do as tech debt once the alpha release is stable.

Reply:
Sounds good. Could add a TODO for future reference.